testing/smoke-ide-test/build.gradle.kts

```diff
@@ -81,6 +82,7 @@ tasks {
         group = "Verification"
         maxParallelForks = 1
         systemProperties["org.gradle.integtest.executer"] = "forking"
+        systemProperties["ide.starter.path"] = ideStarterDir.map { it.asFile.absolutePath }.get()
```

Comment:
❓ Isn't it going to break caching? You need to use the argument provider with normalization, like described [there](https://docs.gradle.org/current/userguide/caching_java_projects.html#dealing_with_file_paths)

Reply:
Wow, I was suspecting that there is a recommended way of a hundred loc instead of a single line 😀 
Thanks!